tutorials/hyperspectral/Work-With-Hyperspectral-Data-In-R.md

```diff
@@ -60,7 +60,7 @@ preferably, RStudio loaded on your computer.
 The electromagnetic spectrum is composed of thousands of bands representing 
 different types of light energy. Imaging spectrometers (instruments that collect 
 hyperspectral data) break the electromagnetic spectrum into groups of bands that 
-support classification of objects by their spectral properties on the earth's 
+support classification of objects by their spectral properties on the Earth's 
```

Comment:
I'm confused by several changes in the .md that don't appear as corresponding changes in the .Rmd.  But the Review layout is making it challenging to track these down.  I just want to make sure any change in the .md is also in the .Rmd.